pkgs/development/libraries/opensubdiv/default.nix

```diff
@@ -50,7 +50,6 @@ stdenv.mkDerivation rec {
     [ "-DNO_TUTORIALS=1"
       "-DNO_REGRESSION=1"
       "-DNO_EXAMPLES=1"
-      "-DNO_METAL=1" # don’t have metal in apple sdk
```

Comment:
Let's maybe make this explicit as the other options, `(cmakeBool "NO_METAL" (!stdenv.isDarwin))`